### PR TITLE
[docs] Add document for X_VCPKG_APPLOCAL_DEPS_INSTALL

### LINKED ISSUE #16182: wxWidgets package does not copy dlls to build folder
**Describe the bug**
When using vcpkg with cmake and wxWidgets all dlls required by the application are not copied to the build directory. This makes the aplication fail to start.

**Environment**
- OS: Windows

### PR DESCRIPTION
**Pull request by JackBoosY**
Waiting for #16722 merge.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -276,6 +277,15 @@ This will still allow people to not use vcpkg,
 by passing the `CMAKE_TOOLCHAIN_FILE` directly,
 but it will make the configure-build step slightly easier.
 
+### Install dependencies automatically
```
Comment (autoantwort) [inline README.md:280]: Which dependencies? Do you mean manifest mode?

File: README.md [hunk 2]
```diff
@@ -276,6 +277,15 @@ This will still allow people to not use vcpkg,
...
+
+Add the option `X_VCPKG_APPLOCAL_DEPS_INSTALL` and set it to `ON`.
+Then you can install the custom target and install the dependencies of the target at the same time.
```
Comment (autoantwort) [inline README.md:283]: Soory I don't understand what you mean

Comment (pratikpc) [reply README.md:283]: Is this only for dependencies specified in vcpkg.json or would it also copy DLLs for user-defined shared libraries?

Comment (JackBoosY) [reply README.md:283]: Not for manifest, this corresponds to the automatic copy dependency function in integrated msbuild.

Comment (autoantwort) [reply README.md:283]: Does this work on every platform or is this windows only?

Comment (JackBoosY) [reply README.md:283]: @autoantwort Seems for each platform.

**Comment by JackBoosY** [comment]
Fixes https://github.com/microsoft/vcpkg/pull/16557.

**Comment by JackBoosY** [comment]
Close this temporary.

**Pull request closed without merging**